Engine/ScriptAnalyzer.cs

```diff
@@ -1890,6 +1853,51 @@ private IEnumerable<DiagnosticRecord> AnalyzeFile(string filePath)
                 return null;
             }
 
+            // short-circuited processing for a help file
+            // no parsing can really be done, but there are other rules to run (specifically encoding).
+            if ( Regex.Matches(Path.GetFileName(filePath), @"^about_.*help.txt$", RegexOptions.IgnoreCase).Count != 0)
```

Comment:
Do you think it would be better in terms of performance to revert back to using `StartsWith` and `EndsWith` as it did before?

Reply:
it is slightly better in the quick check i did (.07 milliseconds), but I thought this was *much* easier to read.

Reply:
Ok, that's negligible then.